From 7a0583892228738bea20029dab74a0e319c390fa Mon Sep 17 00:00:00 2001
From: Jeremy Cline <jcline@redhat.com>
Date: Tue, 23 Jul 2019 15:18:35 +0000
Subject: [PATCH] [scsi] hpsa: modify hpsa driver version

Rebase for the new version bump with 5.3-rc1

	Message-id: <20190520221930.19040-1-labbott@redhat.com>
	Patchwork-id: 259214
	O-Subject: [ARK INTERNAL PATCH] [scsi] hpsa: modify hpsa driver version
	Bugzilla:
	RH-Acked-by: Rob Evers <revers@redhat.com>
	RH-Acked-by: Don Zickus <dzickus@redhat.com>

	Rebase for the new version bump with 5.2-rc1

		Message-id: <1524750809-43700-3-git-send-email-jszczype@redhat.com>
		Patchwork-id: 211574
		O-Subject: [RHEL8 e-stor PATCH 2/2] hpsa: modify hpsa driver version
		Bugzilla: 1471185
		RH-Acked-by: Maurizio Lombardi <mlombard@redhat.com>
		RH-Acked-by: Tomas Henzl <thenzl@redhat.com>

		Modify driver version to reflect changes to driver. The old
		PCI IDs for smartarrays that had used cciss driver were removed.
		Add the -RH1 to reflect that.

		Acked-by: Don Brace <don.brace@microsemi.com>
		Signed-off-by: Herton R. Krzesinski <herton@redhat.com>

	Signed-off-by: Laura Abbott <labbott@redhat.com>

Signed-off-by: Jeremy Cline <jcline@redhat.com>
---
 drivers/scsi/hpsa.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/scsi/hpsa.c b/drivers/scsi/hpsa.c
index 43a6b5350775..b08d83c533ee 100644
--- a/drivers/scsi/hpsa.c
+++ b/drivers/scsi/hpsa.c
@@ -60,7 +60,7 @@
  * HPSA_DRIVER_VERSION must be 3 byte values (0-255) separated by '.'
  * with an optional trailing '-' followed by a byte value (0-255).
  */
-#define HPSA_DRIVER_VERSION "3.4.20-170"
+#define HPSA_DRIVER_VERSION "3.4.20-170-RH1"
 #define DRIVER_NAME "HP HPSA Driver (v " HPSA_DRIVER_VERSION ")"
 #define HPSA "hpsa"
 
-- 
2.21.0

